tmt/steps/discover/fmf.py

```diff
@@ -27,6 +28,26 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin):
             path: /fmf/root
             test: /tests/basic
             filter: 'tier: 1'
+
+    It is also possible to limit tests only to those that have changed in git
+    since a given revision. This can be particularly useful when testing
+    changes to tests themselves (e.g. in a pull request CI).
+
+    Related config options (all optional):
+    * modified_only - set to True if you want to filter modified tests
```

Comment:
The reason I went with underscores is that `argparse` (and apparently `click`, too) automatically converts `--foo-bar` to key `foo_bar` so it can be accessed as an attribute after parsing the args (both `args.foo_bar` and `args['foo_bar']` are then possible). To get dashes there, you'd need to convince `click` somehow to preserve the dashes in the key name (maybe that's easy, I don't know).

Reply:
Yeah, I understand. We can slightly update `Common.opt()` to automatically convert requests for dashed options to underscores. I guess that should relatively easily cover this.